docs/user/nifi/processors.rst

```diff
@@ -79,6 +80,12 @@ All types of input processors have some common configuration parameters for cont
 | ``Identifying Attribute``     | When using a modifying writer, the attribute used to uniquely identify the feature.     |
 |                               | If not specified, will use the feature ID                                               |
 +-------------------------------+-----------------------------------------------------------------------------------------+
+| ``Schema Compatibility``      | Controls how differences between the configured schema and the existing schema in the   |
+|                               | data store (if any) will be handled. ``Existing`` will use the existing schema and drop |
+|                               | any additional fields in the configured schema. ``Update`` will update the existing     |
+|                               | schema to match the configured schema. ``Exact`` requires the configured schema to      |
+|                               | match the existing schema.                                                              |
```

Comment:
Is it easy to add break out bullets for each of these options inside this table?

Reply:
ah, good question. i think we do that other places, so ti should be possible. i'll try to give it a shot this afternoon.